app/main.py

```diff
@@ -17,8 +17,11 @@ def __init__(
         self.count_of_ratings = count
 
     def calculate_washing_price(self, car: Car) -> float:
-        return round(car.comfort_class * (self.clean_power - car.clean_mark)
-                     * self.average_rating / self.distance_from_city_center, 1)
+        return round(car.comfort_class
+                     * (self.clean_power
```

Comment:
Not it is formatted in right way, BUT.
In order to write more readable and maintainable code you should divide huge formula into small parts with using additional variables

Reply:
Thanks, now I get it